sql/src/main/java/io/crate/breaker/ConstSizeEstimator.java

```diff
@@ -40,7 +44,16 @@ public long estimateSize(@Nullable Object value) {
     }
 
     @Override
-    public long estimateSize(@Nullable Object oldValue, @Nullable Object newValue) {
-        return estimateSize(newValue);
+    public long estimateSizeDelta(@Nullable Object oldValue, @Nullable Object newValue) {
```

Comment:
what the heck goes on here? ;)
add some explanation for poor, stupid guys like me.